includes/Form.class.php

```diff
@@ -165,6 +165,18 @@ function process($params) {
 		}
 		$email = $_POST["email"];
 		$phone = isset($_POST["phone"]) ? $_POST["phone"] : "";
+		$lists = (isset($_POST["nlbox"]) && $_POST["nlbox"]) ? $_POST["nlbox"] : array();
+		if (isset($_SESSION["image_random_value"])) {

```

Comment:
yeah i get that part, but that still doesnt address the issue. if u just point a script to submit a form, the script didnt have previous session, then this check will be bypassed. can we solve that?